docs/5.0/preview/upcoming-releases.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Teleport Upcoming Releases
+description: A timeline of teleport upcoming releases.
+---
+
+# Upcoming Teleport Releases
+
+The teleport team delivers a new major release every 3 months.
+
+## Teleport 6.0 "San Diego"
+
+The team open sources role based access controls, implements Database Access, Session Termination
+and Access Workflows UI.
+
+
+### Release Schedule
+
+|Version              | Date              | Description
+|---------------------|-------------------|---------------------------
+|First alpha          | Jan 15th, 2021    | Good for testing and demos.
+|First beta           | Feb 1st, 2021     | Deploy on staging.
+|Release              | March 1st, 2021   | Good to go for production.
+
+### Features
+
+You can find the full list of fixes and features in the
+[Github milestone](https://github.com/gravitational/teleport/milestone/33).
+
+|Feature                              | Editions          | Description
+|-------------------------------------|-------------------|-----------------------------------
+| Database Access                     | All               | SSO into PostgreSQL or MySQL. Use AWS Aurora RDS or on-premises. Read [more here](./database-access.md).
+| OSS RBAC                            | OSS               | Open source role based access controls. Check out the [design doc](https://github.com/gravitational/teleport/blob/master/rfd/0007-rbac-oss.md) and [issue 4136](https://github.com/gravitational/teleport/issues/4136).
+| Terraform Provider                  | All               | Configure teleport without UI using Terraform provider. More details [here](https://github.com/gravitational/teleport-plugins/projects/3#card-49866475).
+| Dual Authorization Workflows        | Enterprise, Cloud | Request multiple users to review and approve access requests. Find out more in [issue 5007](https://github.com/gravitational/teleport/issues/5007).
+| U2F for Kubernetes and SSH sessions | All               | Adds an option to authorize with 2nd factor when connecting to a node/k8s cluster. Details in [issue 3828](https://github.com/gravitational/teleport/issues/3878).
+| Access Workflows UI                 | Enterprise, Cloud | Review access requests and assume roles in the UI. Some mockups are in [issue 4937](https://github.com/gravitational/teleport/issues/4937).
+| Client libraries and API            | All               | Use Golang to create access workflows. Review the [design doc](https://github.com/gravitational/teleport/pull/4746) and [issue 4763](https://github.com/gravitational/teleport/issues/4763).
+
+## Semantic Versioning
+
+Teleport follows [semantic versioning](https://semver.org/) for pre-releases and releases.
+
+**Pre-releases**
+
+Pre-releases have suffixes `-alpha`, `-beta` and `-rc`.
```

Comment:
In the RFD, I proposed only using `-dev` and `-rc.X` suffixes: https://github.com/gravitational/teleport/blob/52fdfa90ffba6cb10714718193aa945a95a50d04/rfd/0012-teleport-versioning.md#details
There wouldn't be any `-alpha` or `-beta`.

If we agree on the RFD, this should be updated to match.

Reply:
Sorry, I missed that part. Let's do `alpha` and `beta` please. We have to communicate differences in expectation.